apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -241,35 +240,62 @@ export class PoliceService {
   async updatePoliceCase(
     user: User,
     caseId: string,
-    caseType: CaseType | IndictmentSubtype,
+    caseType: CaseType,
     caseState: CaseState,
-    courtRecordPdf: string,
     policeCaseNumber: string,
-    defendantNationalIds?: string[],
-    caseConclusion?: string,
+    defendantNationalId: string,
+    validToDate: Date | undefined,
```

Comment:
validToDate?: Date, maybe like this?

Reply:
True, but then I need to add it to the end of the parameter list :(  I'm going to leave it like this for now.

Reply:
I am making further changes to this code and may modify this with that change.